README.md

```diff
@@ -835,6 +835,13 @@ Troubleshooting
 
 To get help, follow the instructions in the [shared Troubleshooting document](https://github.com/GoogleCloudPlatform/gcloud-common/blob/master/troubleshooting/readme.md#troubleshooting).
 
+### Proxy (Experimental)
```

Comment:
I don't think this should be a subsection of Troubleshooting. I think we should document how http proxying and grpc proxying are done as subsections of a single top-level section. Then, each API should state which transport it uses, so that users can know which proxying solution needs to be used.